examples/sri-1/components/workgroup/workgroup/workgroup.interface.ts

```diff
@@ -0,0 +1,34 @@
+import { Workstep } from '../workstep/workstep';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+import { Workflow } from '../workflow/workflow';
+
+export interface IWorkgroup {
+    id: string; // TODO: Add uuid after #491
+    name: string;
+    administrator: BpiSubject[];
+    securityPolicy: Security;
+    privacyPolicy: Privacy;
+    participants: BpiSubject[];
+    worksteps: Workstep[];
+    workflows: Workflow[];
+
+    addParticipant(bpiSubject: BpiSubject);
```

Comment:
@Kasshern why not use array here to allow for addition ofr more than one participant rather than have another object?

Reply:
Altered to use array.